test/roby_app/spawn_server/test_spawn_server.rb

```diff
@@ -108,6 +108,7 @@ def upload_testfile
                     ftp.get("#{@temp_serverdir}/testfile")
                 end
             end
+            File.delete("testfile")
```

Comment:
`@temp_serverdir` is created with `make_tmpdir`. It is therefore cleaned up automatically and you don't need to delete the file manually.